targets/DeployDependencies.proj

```diff
@@ -25,20 +26,32 @@
   <PropertyGroup>
     <RuntimeProjectJson>$(MSBuildThisFileDirectory)\runtime.project.json</RuntimeProjectJson>
     <RuntimeProjectLockJson>$(MSBuildThisFileDirectory)\runtime.project.lock.json</RuntimeProjectLockJson>
-    <RestoreRuntimePackagesCommand>"$(NuGetToolPath)" restore $(RuntimeProjectJson)</RestoreRuntimePackagesCommand>
+    <RestoreRuntimePackagesCommand>$(NugetRestoreCommand) $(NuGetPackageInstallOptions) $(RuntimeProjectJson)</RestoreRuntimePackagesCommand>
```

Comment:
It *shouldn't* be required to restore these packages to our local packages directory.  The idea is to restore them to your machine/user NuGet cache, then use `ResolveNuGetPackageAssets` to copy them locally.  I believe that this should be robust now after my change to run the restore twice to work around the NuGet bug.